derive_builder_core/src/builder.rs

```diff
@@ -198,17 +203,25 @@ impl<'a> ToTokens for Builder<'a> {
                 impl #impl_generics #builder_ident #ty_generics #where_clause {
                     #(#functions)*
                     #deprecation_notes
-                }
 
-                impl #impl_generics ::derive_builder::export::core::default::Default for #builder_ident #ty_generics #where_clause {
-                    fn default() -> Self {
+                    fn create_empty() -> Self {
```

Comment:
I've added override support for this. It would be nice to have it check for a collision with a field name, but that may have to be out of scope for now.